CMakeLists.txt

```
@@ -206,6 +206,11 @@ install(
     DESTINATION "${CMAKE_INSTALL_BINDIR}"
     PERMISSIONS ${EXECUTABLE_PERMISSIONS}
 )
+install(
+    FILES "${CMAKE_CURRENT_SOURCE_DIR}/script/generate_needle_previews"
```

It would be nicer if the name of the script would follow our pattern for new scripts:
```suggestion
    FILES "${CMAKE_CURRENT_SOURCE_DIR}/script/os-autoinst-generate-needle-previews"
```